web/src/app/lists/FlatList.js

```diff
@@ -121,7 +123,7 @@ export default class FlatList extends React.PureComponent {
           <ListItemText
             disableTypography
             secondary={
-              <Typography variant='caption'>
+              <Typography data-cy='empty-message-cptn' variant='caption'>
```

Comment:
What do you think of just `empty-message` or `list-empty-message` to leave out the variant bit (just focus on function)? At any rate, `cptn` makes me think captain :)